.github/CONTRIBUTING.md

```diff
@@ -217,6 +223,26 @@ from Transifex to the source code and vice versa.
 
 Please have a look at the specific [FAQ for translators](https://github.com/Cockatrice/Cockatrice/wiki/Translation-FAQ).
 
+
+# Release Management #
+
+### ToDo list around new main releases ###
```

Comment:
I agree on `TODO`, but the font is what makes it look like `reLeases`; it's actually just `releases`.